apps/web/src/app/stream/sandbox/[sandboxId]/page.tsx

```diff
@@ -0,0 +1,53 @@
+import { Suspense } from 'react'
+import { createClient } from '@supabase/supabase-js'
+import MuxPlayer from '@mux/mux-player-react'
+
+const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL!
+const supabaseServiceKey = process.env.SUPABASE_SERVICE_ROLE_KEY!
+
+const supabase = createClient(supabaseUrl, supabaseServiceKey)
+
+interface SandboxStream {
+  sandboxId: string
+  playbackId: string
+}
+
+async function fetchStream(sandboxId: string): Promise<SandboxStream | null> {
+  const { data, error } = await supabase
+    .from('sandbox_streams')
+    .select('playback_id')
+    .eq('sandbox_id', sandboxId)
+    .single()
+
+  if (error || !data) {
+    return null
+  }
+
+  return { sandboxId, playbackId: data.playback_id }
+}
+
+export default async function StreamPage({ params }: { params: { sandboxId: string } }) {
+  const stream = await fetchStream(params.sandboxId)
```

Comment:
No security?

Reply:
Discussed this with @ValentaTomas. Streams are public so reading them should be public. Maybe we could allow creating them only if you're sending an E2B API key @ValentaTomas ?

Reply:
Creating them is behind API key+sandbox ID.

For access, we want there to be just a link you can follow. The most straightforward solution here then could be that we also pass some per sandbox stream secret when we create the sandbox that is then added to the link returned in SDK in query params.

That could offer more flexibility in the future, because right now the sandbox ID also serve as the secret you use for connecting.

Reply:
To future-proof this, it might be worth adding and passing such a field so we don't face compatibility problems later.

Reply:
Done